### ISSUE TITLE
Facing "CUDA out of memory" when running the "detect_dual.py"

### ISSUE THREAD

**Issue by WindRise29** [WongKinYiu/yolov9#431]
When i running the "detect_dual.py", i facded the problem of "CUDA out of memory" as shown in Figure. However, at the same time, the same device, i can run the detect file of yolov8 successfully. (The detected date are the same.)  What causes the situation?
![yolov8](https://github.com/WongKinYiu/yolov9/assets/130849481/c943cee9-11b9-4970-ab64-378cf7aeefce)
![yolov9](https://github.com/WongKinYiu/yolov9/assets/130849481/ac862584-cdcf-4f28-9243-58f510aeec9f)

**Comment by jdiaz97** [comment]
Your image size is quite big, you are gonna have to reduce your batch, try with 1 at least

**Comment by WindRise29** [comment]
> Your image size is quite big, you are gonna have to reduce your batch, try with 1 at least
yeah, i'm gonna try it, and also, i did not delete the auxiliary branch, maybe it can also increase the computing pressure. i m also working on that. Thank you.

**Comment by khanhjt** [comment]
try again bro, when your laptop shutdown 😵‍💫

**Comment by rlewkowicz** [comment]
No, something is wrong. Infact lowering batch size INCREASES the gpu memory on my linux laptop. This same laptop and tool chain on windows has no issues. My desktop has no issues. I'll try to track this down if I have the time, but something is broken somewhere

**Comment by rlewkowicz** [comment]
Nope, even the desktop is having issues now although I never tried this tool chain on Linux. Ultralytics works etc

**Comment by rlewkowicz** [comment]
`conda install pytorch==2.2.1 torchvision==0.17.1 torchaudio==2.2.1 pytorch-cuda=12.1 -c pytorch -c nvidia`

Its the torch version i think version. https://pytorch.org/get-started/previous-versions/#v221

**Comment by rlewkowicz** [comment]
I was trying to get training started before running out the door. I still dont know if this is right
